lib/modules/manager/mix/artifacts.spec.ts

```diff
@@ -260,9 +285,9 @@ describe('modules/manager/mix/artifacts', () => {
     // elixir
     getPkgReleases.mockResolvedValueOnce({
       releases: [
-        { version: 'v1.8.2' },
-        { version: 'v1.13.3' },
-        { version: 'v1.13.4' },
+        { version: '1.8.2' },
+        { version: '1.13.3' },
+        { version: '1.13.4' },
```

Comment:
please revert, those versions will be `v` prefixed

https://github.com/renovatebot/renovate/blob/9b4e5150248a6b0026acf9c73bb00a295cba4c48/lib/util/exec/containerbase.ts#L61-L65

https://github.com/elixir-lang/elixir/releases/tag/v1.18.1